### ISSUE TITLE
Accessibility Fixes

### ISSUE THREAD

**Issue by DanielMiller-agile6** [agilesix/AgileSix.com#18]
I ran some quick accessibility tests over the agile6.com site (using Axe Chrome plugin) and found a few issues.  There aren't many serious or critical failures, but most of these should be quick fixes.  We're also missing [skip nav links](https://webaim.org/techniques/skipnav/), which are helpful for screen readers and keyboard navigation. As a government contracting company, it might be good to show that we practice what we preach regarding 508 compliance :).

First steps would be to resolve issues found from these tests, but we could delve deeper in the future with a more robust accessibility audit and automated axe-core tests as part of our build and deploy process.

Here are a few screenshots of errors I found:

![Screen Shot 2020-03-13 at 2 33 56 PM](https://user-images.githubusercontent.com/54035677/76661679-4b5abb00-6539-11ea-8385-4445c74e649c.png)

![Screen Shot 2020-03-13 at 2 36 18 PM](https://user-images.githubusercontent.com/54035677/76661703-56ade680-6539-11ea-9ac8-40c0e72e0582.png)

![Screen Shot 2020-03-13 at 2 37 18 PM](https://user-images.githubusercontent.com/54035677/76661707-5a416d80-6539-11ea-9e6a-afd44cf899cb.png)

**Comment by ElijahLynn** [comment]
Thanks for reporting this, tracking for now until have more time to help out!
